analytics-samples/analytics-sample/src/main/java/com/segment/analytics/sample/SampleApp.java

```diff
@@ -25,82 +25,99 @@
 
 import android.app.Application;
 import android.util.Log;
+
 import com.segment.analytics.Analytics;
 import com.segment.analytics.Middleware;
 import com.segment.analytics.ValueMap;
 import com.segment.analytics.integrations.BasePayload;
 import com.segment.analytics.integrations.TrackPayload;
+import com.segment.jsmiddleware.EdgeFunctionMiddleware;
+
 import io.github.inflationx.calligraphy3.CalligraphyConfig;
 import io.github.inflationx.calligraphy3.CalligraphyInterceptor;
 import io.github.inflationx.viewpump.ViewPump;
 
 public class SampleApp extends Application {
 
-  // https://segment.com/segment-engineering/sources/android-test/settings/keys
-  private static final String ANALYTICS_WRITE_KEY = "HO63Z36e0Ufa8AAgbjDomDuKxFuUICqI";
+    // https://segment.com/segment-engineering/sources/android-test/settings/keys
+    private static final String ANALYTICS_WRITE_KEY = "1vCwH2CEddxqO0nsk7rKezv0gzFNlJGH";
```

Comment:
Should we reset this write key?

Reply:
Maybe we should do something generic key with a TODO above it for when people want to use the sample app they'll know where to add their own key easily via the IDE